### ISSUE TITLE
Some questions about loss

### ISSUE THREAD

**Issue by xiaocc612** [LiuShuaiyr/UAVMOT#14]
1. Previous work such as Fairmot only used cross-entropy loss in the reID branch, while this work uses triplet loss, how much gain does this bring?
2. In a mini-batch, when there is only one identity for a class, such as tricycle, how do you handle it when calculating the triplet loss (running the existing code will report an error)? Do you skip it and not calculate it?

Thanks in advance.

**Comment by xiaocc612** [comment]
> 1. Previous work such as Fairmot only used cross-entropy loss in the reID branch, while this work uses triplet loss, how much gain does this bring?
> 2. In a mini-batch, when there is only one identity for a class, such as tricycle, how do you handle it when calculating the triplet loss (running the existing code will report an error)? Do you skip it and not calculate it?
> 
> Thanks in advance.

In my experiments, I found that directly using FocalLoss to supervise heatmap is better than the GBF loss proposed in your paper. Is it necessary to set hyperparameters for EQLv2? The existing default settings do not work.
@LiuShuaiyr

**Comment by ChiachengZhang** [comment]
> 你好，关于这个代码我有些问题，能私下请教一下吗，我的微信：wwddzjc
